modules/core/src/v2/coins/eos.ts

```diff
@@ -408,4 +438,268 @@ export class Eos extends BaseCoin {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Validate a public or private key
+   * If passphrase is provided, try to decrypt the key with it
+   * @param key
```

Comment:
types?

Reply:
Typedoc will take them from the typescript types, so no need.